communication/data_logging.hpp

```diff
@@ -112,8 +104,10 @@ typedef struct
 	
 	uint32_t sys_id;							///< the system ID
 	
+	Console<File>* console;						///< The pointer to the console containing the file to write data to
```

Comment:
I don't think the standard files should go inside the board, more in central data